src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py

```diff
@@ -390,10 +391,10 @@ def carp_master(self, fobj, ifname, vhid, event, user_override, forcetakeover):
                 self.logger.warn('Acquired failover master lock')
                 self.logger.warn('Starting fenced')
                 if not user_override and not fasttrack and not forcetakeover:
-                    error, output = run('LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced')
+                    error, output = run(f'LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced -ed {fobj["boot_disks"]}')
                 else:
                     error, output = run(
-                        'LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced --force'
+                        f'LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced --force --ed {fobj["boot_disks"]}'
```

Comment:
-ed or --ed? does it matter?

Reply:
it definitely matters, so a typo on my part. fixed.